runtime/lua/vim/diagnostic.lua

```diff
@@ -643,6 +643,13 @@ function M.set(namespace, bufnr, diagnostics, opts)
   vim.api.nvim_command("doautocmd <nomodeline> User DiagnosticsChanged")
 end
 
+--- List current diagnostic namespaces
+---
+---@return table A list of active diagnostic namespaces |vim.diagnostic|.
+function M.list_namespaces()
```

Comment:
Maybe `get_namespaces()` instead to be more consistent with e.g. `nvim_get_namespaces()`?

Reply:
done